utils/statediff/src/main.rs

```diff
@@ -1,68 +1,86 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
+use clap::Parser;
 use directories::ProjectDirs;
 use forest_db::db_engine::{db_path, open_db, DbConfig};
-use structopt::StructOpt;
 
 use cid::Cid;
 use forest_statediff::print_state_diff;
 
-/// Examine the state delta
-#[derive(StructOpt)]
-pub struct ChainCommand {
-    /// The previous CID state root
-    pre: Cid,
-    /// The post CID state root
-    post: Cid,
-    /// The name of the chain
-    #[structopt(short, long, default_value = "mainnet")]
-    chain: String,
-    /// The depth at which IPLD links are resolved
-    #[structopt(short, long)]
-    depth: Option<u64>,
-}
+// /// Examine the state delta
+// #[derive(clap::Subcommand)]
```

Comment:
Is this part incomplete or to be removed?

Reply:
Removed